### PR TITLE
Fixed animation color bug for Safari.

### PR DESCRIPTION
**Pull request by nagnibedamihai**
Fixed a bug on Safari browser: the animation layer was rendered grey instead of white.

### CODE REVIEW

**Comment by zalog** [comment]
Nice catch, tnk!

File: src/scss/_layout.scss [hunk 1]
```diff
@@ -25,7 +25,7 @@
     width: 500%;
     margin-left: -250%;
     animation: phAnimation .8s linear infinite;
-    background: linear-gradient(to right, transparent 46%, rgba($ph-bg, .35) 50%, transparent 54%) 50% 50%;
+    background: linear-gradient(to right, rgba(255,255,255,0) 46%, rgba($ph-bg, .35) 50%, rgba(255,255,255,0) 54%) 50% 50%;
```
Comment (zalog) [inline src/scss/_layout.scss:28]: Can you please confirm that it works with:
`linear-gradient(to right, rgba($ph-bg, 0) 46%, rgba($ph-bg, .35) 50%, rgba($ph-bg, 0) 54%) 50% 50%;`?

File: src/scss/_layout.scss [hunk 2]
```diff
@@ -53,7 +53,7 @@
     margin-bottom: $ph-spacer;
   }
   .empty {
-    background-color: transparent;
+    background-color: rgba(255,255,255,0);
```
Comment (zalog) [inline src/scss/_layout.scss:56]: Same here, `rgba($ph-bg, 0)`

File: src/scss/_layout.scss (hunk 1 above)
Comment (nagnibedamihai) [reply src/scss/_layout.scss:28]: works

File: src/scss/_layout.scss (hunk 2 above)
Comment (nagnibedamihai) [reply src/scss/_layout.scss:56]: works

**Comment by zalog** [comment]
Great! I think this is it.
I missed that on this pr... you can make a new one if you want.

**Pull request merged**